src/App.cpp

```diff
@@ -228,7 +228,7 @@ int MyApp::OnExit()
         m_luaLog->Close();
         // Check to see if we have lua messages
         if (m_luaMessages) {
-            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), (const wxChar *)GetLuaLogFilename().c_str());
+            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), GetLuaLogFilename().c_str().AsChar());

```

Comment:
Yes, wx_str() on Mac results in only the first character being printed, unless I change the format string to %ls. But that seems iffy to me, as wx_str() seems like it could return char* which would break this assumption.

So the safest thing (I think?) seems to be to use mb_str() and change the format string to %hs as mb_str() is guaranteed to be narrow in all cases. An alternative would be to use wc_str() and use %ls. I'm still a bit worried about what happens if more obscure characters show up in these messages, though.

What do you think now?